README.md

```diff
@@ -8,6 +8,91 @@ An implementation of [Amazon Ion Schema](http://amzn.github.io/ion-schema) in Ru
 
 **This package is considered experimental, under active/early development, and the API is subject to change.**
 
+## Getting Started
+
+The following rust code sample is a simple example of how to use this API. The `my_int_type` is an integer ion type. 
+A sample schema `my_schema.isl` can be loaded using `ion-schema-rust` as following:
+
+### Example schema `my_schema.isl`
```

Comment:
Consider moving some of the exposition about the schema itself into the schema subsection.

```suggestion
## Getting Started

The following rust code sample is a simple example of how to use this API.

### Example schema `my_schema.isl`

This file (`my_schema.isl`) defines a new type (`my_int_type`) based on Ion's `int` type.
```
The section starting on line 32 (where the schema is loaded/used) already says that's what it's doing, so I removed this bit:
> A sample schema `my_schema.isl` can be loaded using `ion-schema-rust` as following: